Thank you screen

DimaPylyp opened the pull request:
Created Thank You Screen and added needed constants.

DimaPylyp committed:
-Created ThankYouVC for Thank You Screen.
- Added Constants that are used in ThankYouVC

alexxcheung commented:
Thank you

Ericordo commented on LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift:
```diff
@@ -0,0 +1,167 @@
+    private lazy var confettiView = SAConfettiView(frame: view.bounds)
+    
+    private let heartIcon : UIImageView = {
+        
```
Remove space

Ericordo commented on LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift:
```diff
@@ -0,0 +1,167 @@
+    
+    private let heartIcon : UIImageView = {
+        
+        let icon = UIImageView(image: UIImage(named: Images.heartIcon)!.withAlignmentRectInsets(UIEdgeInsets(top: -10, left: 0, bottom: 0, right: 0)))
```
make it in two lines
let icon = UIImageView()
icon.image = UIImage blablabla

Ericordo commented on LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift:
```diff
@@ -0,0 +1,167 @@
+        let icon = UIImageView(image: UIImage(named: Images.heartIcon)!.withAlignmentRectInsets(UIEdgeInsets(top: -10, left: 0, bottom: 0, right: 0)))
+        icon.contentMode = .scaleAspectFit
+        
+        icon.translatesAutoresizingMaskIntoConstraints = false
```
This is supposed to be in addConstraints method

Ericordo commented on LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift:
```diff
@@ -0,0 +1,167 @@
+            icon.heightAnchor.constraint(equalToConstant: 60)
+        ])
+        
+//        icon.layer.borderWidth = 1
```
Delete commented code

Ericordo commented on LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift:
```diff
@@ -0,0 +1,167 @@
+        label.textAlignment = .left
+        label.font = UIFont.boldSystemFont(ofSize: 22)
+        label.text = LabelStrings.thankYou
+//        label.layer.borderWidth = 1
```
Delete commented code

Ericordo commented on LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift:
```diff
@@ -0,0 +1,167 @@
+        return stackView
+    }()
+    
+    private let contentView = UIView()
```
Bad copy pasting

Ericordo commented on LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift:
```diff
@@ -0,0 +1,167 @@
+    
+    @objc private func didTapContinue() {
+        print("tap")
+        defaults.set(true, forKey: Keys.onboardingComplete)
```
No, this is not onboarding

Ericordo commented on LetsDinnerV2 MessagesExtension/Controller/ThankYouVC/ThankYouViewController.swift:
```diff
@@ -0,0 +1,167 @@
+//
+
+import UIKit
+import SafariServices
```
Why SafariServices?

DimaPylyp committed:
-Deleted lines that were not used
-Deleted commented code
-Edit Icon initialisation
-Moved icon.translatesAutoresizingMaskIntoConstraints to addConstraints method

DimaPylyp committed:
-Updated constraints in ThankYouVC
